tutorial/creating_packages/other_packages/tool_requires/conanfile.py

```diff
@@ -0,0 +1,21 @@
+import os
+import stat
+from conan import ConanFile
+from conan.tools.files import save
+
+
+class my_tool(ConanFile):
+    name = "my_tool"
+    version = "1.0"
+    package_type = "application"
+
+    settings = "os", "arch"
+
+    def package(self):
```

Comment:
Maybe a bit forced example? In real cases, the .bat and .sh will be user files, that are `exports_sources``?

I see it is more convenient this was for the example, maybe enough to add a comment explaining "this is just an example, typically these shell scripts come from user sources"

Reply:
In general, I thought that the important thing was to explain how to package an executable, no matter if it is `sh` or `exe`, I don't care how the operating system runs the executable (with +x permissions). But clearly, it is causing you confusion.

Reply:
Ah, now I see your intent, you were not really trying to implement the "I package my build scripts", but to simply fake an executable. In that sense, I agree with Carlos it would be better to build a real executable to use it. Otherwise, I would prefer that the shell scripts are from the user folder, not generated in the recipe.